content/links.md

```diff
@@ -227,13 +227,76 @@ instance_groups:
       conn: {as: db_conn}
 ```
 
+### Explicit Linking {: #explicit }
+Explicit linking is defined as specifying your consumer in your deployment manifest. The consumer will be matched on both name and type.
+
+##### Providers
+Explicitly defined providers which can be both explicitly and implicitly consumed, can have the following optional properties:
+
+* **as** [String]: Overrides the name of the provider defined in the release spec. Example:
+```yaml
+instance_groups:
+- name: my_instance_group
+  jobs:
+  - name: my_provider_job
+    provides:
+      bar: {as: new_name}
+```
+* **shared** [Boolean]: *Default is false* Sets whether this provider is consumable from another deployment. See [cross deployment links](#cross-deployment).
+
+Providers that are specified as `nil` will cause your *explicit* consumer to fail resolution.
+
+Deployment Manifest:
+```yaml
+instance_group:
+- name: db_ig
+  jobs:
+  - name: db_job
+    provides:
+      database: nil
+```
+
+##### Consumers
+Explicitly defined consumers can have the following optional properties:
+
+* **from** [String]: Overrides the name of the provider to consume. This should match the name defined in the provider's release spec or the name defined by provider's `as` property in the manifest.
+* **deployment** [String]: The name of the deployment to consume from. If the deployment provided does not exist, it will fail with an error. If the name specified is the same as the current deployment, it will be considered non-cross-deployment.
+* **network** [String]: The name of a network used by the provider. See [custom network linking](#custom-network).
+* **ip_addresses** [Boolean]: *Default is false* Boolean for preferring usage of ip addresses instead of DNS names. In cases such as dynamic network, ip addresses are not available and will still return DNS names. See [dns](dns#links) for more details.
+
+Optional consumers may be defined as `nil` in your deployment manifest to block linking.
```

Comment:
Is "block linking" the phrase to use here? I originally read this as block linking be a new type of linking similar to implicit/explicit. Perhaps this could be rephrased to something like: "Set an optional consumer to `nil` to disable (right word?) that link. See below for an example".

Reply:
Will update to: "Optional consumers may be specified as `nil` in the deployment manifest to block consumption of any providers."